src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.scss

```diff
@@ -0,0 +1,6 @@
+.edit-title-buttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  padding: 10px 0 0 0;
+}
```

Comment:
Should be wrapped/prefixed in the component name. Eg. something like:

```suggestion
.EditTableTitleModal {
  &__edit-title-buttons {
    display: flex;
    flex-direction: row;
    justify-content: space-evenly;
    padding: 10px 0 0 0;
  }
}
```

Reply:
Has it? Still doesn't seem to be wrapped in the BEM pattern shown above in that commit?

Reply:
I changed the namings a bit but it's in this file: https://github.com/sparkletown/sparkle/commit/c194508b129176d4c048ebb810e9ea4c59f60865#diff-2133b9f3a0a87f5ebb579bb760c79ebf5c95f5e93be4712afdcbc35024ad2577

Reply:
Which has ![image](https://user-images.githubusercontent.com/753891/118262555-64be1500-b4f8-11eb-9477-6123ef592948.png)

Suggestion + our standard is to match the name of the component file
![image](https://user-images.githubusercontent.com/753891/118262605-73a4c780-b4f8-11eb-87ac-e067bac01209.png)

Reply:
Which has ![image](https://user-images.githubusercontent.com/753891/118262555-64be1500-b4f8-11eb-9477-6123ef592948.png)

Suggestion + our standard is to match the name of the component file
![image](https://user-images.githubusercontent.com/753891/118262605-73a4c780-b4f8-11eb-87ac-e067bac01209.png)

Reply:
Noted and applied in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a